tests/tests/test-contract-precompiled.ts

```diff
@@ -35,7 +35,7 @@ describeWithMoonbeam("Moonbeam RPC (Contract of precompiled function)", `simple-
         expect(tx_receipt.result.transactionHash).equals('0x44f5a9a11af7d9253f4b34acd87a2f1714610b45c1562783406174e5b70e2c0a');
     });
 
-    it("Call precompiled function with eth_call and ensure the return value is as expected", async function () {
+    it.skip("Call precompiled function with eth_call and ensure the return value is as expected", async function () {
```

Comment:
You've fixexd the ripemd implementation, so this test should no longer be skipped, right?

Reply:
This is a test by same for the deadbeef Precompile (a custom precompile he made, which is not included in the runtime. See the address `0x0000000000000000000000000000000000001000` it is not one of the Eth Precompile)